### ISSUE TITLE
Using RawAxiosRequestHeaders cause compiler error 

### ISSUE THREAD

**Issue by dimkajasons** [axios/axios#6369]
### Describe the bug

I need to create a proper type for axios headers object
I have followed https://github.com/axios/axios/issues/5476 this one - so I can't use AxiosRequestHeaders.

I believe I need to use RawAxiosRequestHeaders. But in case i need to use this type as a generic argument, for example, with `React.createContext` - it cause compiler error 
```
import { RawAxiosRequestHeaders } from 'axios';

export const RequestHeadersContext = createContext<RawAxiosRequestHeaders | undefined>(undefined);
```

```
Exported variable 'RequestHeadersContext' has or is using name 'RawAxiosHeaders' from external module "...node_modules/axios/index" but cannot be named.
```
I guess it's because interface `RawAxiosHeaders` is declared locally.

**Comment by komali2** [comment]
I'm having this issue as well, was there any resolution?

**Comment by nicscthongchai** [comment]
> I'm having this issue as well, was there any resolution?

Try to set your tsconfig.json

```json
{
  "strict": false
}
```

That the only way to fix this for me right now.

**Comment by QusaiSabri** [comment]
I'm having the same issue!

**Comment by chatnarongt** [comment]
> I'm having the same issue!

Do you need `declaration` and `declarationMap` in your project?

If not you can try set them to `false` in you `tsconfig.json`. This might help.
```json
{
  "compilerOptions": {
    "declaration": false,
    "declarationMap": false
  }
}
```

**Comment by jasonsaayman** [comment]
i agree that this can be a TS config error, please try the suggestions and should this still persist please opena a new issue and mention me
